contrib/epee/src/net_ssl.cpp

```diff
@@ -325,6 +329,9 @@ boost::asio::ssl::context ssl_options_t::create_context() const
   {
     case ssl_verification_t::system_ca:
       ssl_context.set_default_verify_paths();
```

Comment:
`SSL_CTX_set_cert_store` replaces the existing store, so a `#else` here maybe? Theres no point to calling `set_default_verify_paths()` in windows if it gets replaced below.